### PR TITLE
json: Replace eslint-plugin-json with eslint-parser-json

### PR DESCRIPTION
**Pull request by edg2s**
This parses the JSON as JS, then applies ESLint style rules,
most importantly: indentation.

Unlike eslint-plugin-json this no longer checks the JSON
is completely valid, but in most cases an invalid JSON
file should cause your build to fail in another way.

### CODE REVIEW

**Comment by edg2s** [comment]
Filed lack of JSON validation as https://github.com/zeitport/eslint-parser-json/issues/4

File: jsdoc.json [hunk 1]
```diff
@@ -47,5 +47,5 @@
 		"jsdoc/require-property-description": "off",
 		"jsdoc/require-property-name": "off",
 		"jsdoc/require-returns-description": "off"
-    }
+	}
```
Comment (edg2s) [inline jsdoc.json:50]: yay

**Comment by edg2s** [comment]
We can't just run eslint-plugin-json in addition to eslint-parser-json as the parser has been changed for all *.json files by the override.

**Comment by jdforrester** [comment]
Interesting. Worth waiting on movement upstream?

**Comment by edg2s** [comment]
Done

**Review by jdforrester — APPROVED** [review]

**Pull request merged**